client/reader/following-edit/style.scss

```diff
@@ -44,6 +44,10 @@
 	float: right;
 }
 
+.is-reader-page .search-card.card {
+	box-shadow: 0 0 0 2px rgba(200, 215, 225, 0.5), 0 1px 2px #e9eff3;
```

Comment:
We try not to use custom colors if possible. From the [coding guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md):

> Don't use custom colors, always utilize what _colors.scss provides. If you have to set a color, use lowercase hex values, and shorten them to their smallest expression (like #aaa).

You can also use `lighten` to modify the preset colors, e.g.:

`border-color: lighten( $gray, 10% );`